cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AdapterHttpHandler.java

```diff
@@ -276,7 +276,7 @@ public void deleteAdapter(HttpRequest request, HttpResponder responder,
       adapterService.removeAdapter(Id.Namespace.from(namespaceId), adapterName);
       responder.sendStatus(HttpResponseStatus.OK);
     } catch (CannotBeDeletedException e) {
-      responder.sendString(HttpResponseStatus.FORBIDDEN, e.getMessage());
+      responder.sendString(HttpResponseStatus.CONFLICT, e.getMessage());
     } catch (NotFoundException e) {
```

Comment:
I'm ok with changing this too - about 7 occurrences of ``NotFoundException`` in ``AdapterHttpHandler``.